src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -54,6 +61,7 @@ public TextEditor(bool shown) : this(Interop.TextEditor.New(), true)
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             SetVisible(shown);
+            initializeTextEditor();
```

Comment:
This is duplicate of line:86

Reply:
The whole initializeText****() functions are not necessary if we decide to do not use null-able property.
I will remove it.